portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -8,13 +8,15 @@
   private final String title;
   private final String author;
   private final long timestamp;
+  private final long rating;
   private final String comment;
   private final long id;
 
-  public Comment(String title, String author, long timestamp, String comment, long id) {
+  public Comment(String title, String author, long timestamp, long rating, String comment, long id) {
```

Comment:
This constructor is getting fairly long and at this point I think should be changed to use the Builder pattern
https://effectivejava.googleplex.com/third-edition#page=31

Reply:
It's not letting me access the document. It says 'You do not have access to the requested resource'

Reply:
But I found some other resources. Don't worry about the access.

Reply:
This is a big change. Should I include it in another PR?

Reply:
Yes, that's a good idea.  I'll approve this one and you can create another one for the builder change.

Reply:
Maybe also add a TODO note to the current PR to indicate you plan to work on it in the future.